.github/workflows/increment-minor.yml

```diff
@@ -0,0 +1,34 @@
+name: Increment Minor
+
+on:
+  workflow_dispatch:
```

Comment:
Can you add https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows as a link of a quick explanation of the events